test/games/strategy/triplea/delegate/LHTRTest.java

```diff
@@ -227,6 +227,48 @@ public Object invoke(final Object proxy, final Method method, final Object[] arg
     // Changed to match StrategicBombingRaidBattle changes
     assertEquals(PUsBeforeRaid - 8, PUsAfterRaid);
   }
+
+  @Test
+  public void testBombingRaid2targets() {
+    final Territory germany = m_data.getMap().getTerritory("Germany");
```

Comment:
It's a work in progress. Was looking for suggestions on how to get it right. 

Can you at least point me to where the xml is picked up from?